packages/chains/src/bsc/plugins.ts

```diff
@@ -4,32 +4,38 @@ const deployedPlugins: DeployedPlugins = {
   "0x10C90bfCFb3D2A7ae814dA1548ae3a7fC31C35A0": {
     market: "0x34ea4cbb464E6D120B081661464d4635Ca237FA7",
     name: "Bomb",
-    docsUrl: "https://www.bomb.farm/#/bsc/vault/bomb-bomb",
+    apyDocsUrl: "https://www.bomb.farm/#/bsc/vault/bomb-bomb",
+    strategyDocsUrl:
+      "https://app.gitbook.com/s/66NW0CnbA4sgMbZnOH21/guides/assets-and-strategies-addresses/binance-smart-chain-bsc/bomb",
   },
   "0x6B8B935dfC9Dcd0754eced708b1b633BF73FE854": {
     market: "0x4cF3D3ca995beEeEd83f67A5C0456A13e038f7b8",
     name: "BTCB-BOMB",
-    docsUrl: "https://www.bomb.farm/#/bsc/vault/bomb-bomb-btcb",
+    apyDocsUrl: "https://www.bomb.farm/#/bsc/vault/bomb-bomb-btcb",
+    strategyDocsUrl:
+      "https://app.gitbook.com/s/66NW0CnbA4sgMbZnOH21/guides/assets-and-strategies-addresses/binance-smart-chain-bsc/bomb",
   },
   "0x3c29e9b0CfE6FfF97f373eAbEADE9475FaC3bd4e": {
     market: "0xf0a2852958aD041a9Fb35c312605482Ca3Ec17ba",
     name: "2brl DotDotLpERC4626",
-    docsUrl: "https://dotdot.finance/#/stake",
+    apyDocsUrl: "https://dotdot.finance/#/stake",
+    strategyDocsUrl:
+      "https://app.gitbook.com/s/66NW0CnbA4sgMbZnOH21/guides/assets-and-strategies-addresses/binance-smart-chain-bsc/jarvis-jfiat-pool",
```

Comment:
These are the private gitbook address and won't work for anyone else

Reply:
Should other url be used for docs link?

Reply:
Yes of course, they should be the docs.midascapital.xyz , not the private gitbook addresses

Reply:
Added public docs urls